packages/starlight/translations/es.json

```
@@ -22,5 +22,8 @@
   "aside.note": "Nota",
   "aside.tip": "Consejo",
   "aside.caution": "Precaución",
-  "aside.danger": "Peligro"
+  "aside.danger": "Peligro",
+  "expressiveCode.copyButtonCopied": "Copiado",
```

Nit: The original text ends with an exclamation mark ("Copied!"), which I added to make it sound a little more affirmative/positive. I'd love this to be present in the translations as well!

---

Something like this @kevinzunigacuellar?

```suggestion
  "expressiveCode.copyButtonCopied": "¡Copiado!",
```

---

¡Sí, señor Chris!